client/blocks/payment-request/branded-buttons.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getStripeServerData } from '../stripe-utils';
+
+export const shouldUseGooglePayBrand = () => {
+	const ua = window.navigator.userAgent.toLowerCase();
+	const isChrome =
+		/chrome/.test( ua ) &&
+		! /edge|edg|opr|brave\//.test( ua ) &&
+		window.navigator.vendor === 'Google Inc.';
+	// newer versions of Brave do not have the userAgent string
+	const isBrave = isChrome && window.navigator.brave;
+	return isChrome && ! isBrave;
+};
+
+const useLocalizedGoogleSvg = ( type, theme, locale ) => {
+	// If we're using the short button type (i.e. logo only) make sure we get the logo only SVG.
+	const googlePlaySvg =
+		type === 'long'
+			? `https://www.gstatic.com/instantbuy/svg/${ theme }/${ locale }.svg`
+			: `https://www.gstatic.com/instantbuy/svg/${ theme }_gpay.svg`;
+
+	const [ url, setUrl ] = useState( googlePlaySvg );
+
+	useEffect( () => {
+		const im = document.createElement( 'img' );
+		im.addEventListener( 'error', () => {
+			setUrl(
+				`https://www.gstatic.com/instantbuy/svg/${ theme }/en.svg`
+			);
+		} );
+		im.src = url;
+	}, [ url, theme ] );
+
+	return url;
+};
+
+export const GooglePayButton = ( { onButtonClicked } ) => {
+	const {
+		theme = 'dark',
```

Comment:
The `light-outline` theme also doesn't work here.

If you want, you can sync [the changes](https://github.com/Automattic/woocommerce-payments/pull/1529/commits/ffb0de4a17cca2ba41cea09e35b9a0404cad50f4) I made in WCPay or simply map `light-outline` to `light`.

Reply:
Made the changes in 7ef1d24f79d4a0f7231f26d008cad6afb77d4365, thanks for the link to the WCPay implementation!

Stripe apparently only has a `light` theme for the GPay buttons, there's no styling applied for `light-outline`. Not sure why, but I decided to keep that theme here. So I just added the necessary margin (to the light theme) and normalized any non-dark theme to `light`.

This is something we may want to change in the future, but I think this is good enough for now.

Reply:
As of c4c794f4ff6a9b9ef16889c59d2c81b1157f52a1 there is now a `light-outline` theme that works for both the Blocks and Shortcode PRBs 🎉